<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="Label_Price">
        <property name="text">
         <string>가격 정보</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_Keyboard">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>Keyboard Price</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEdit_Monitor">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>Monitor Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ButtonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonRead">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
